.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,90 @@
+    id: description
+    attributes: 
+      label: Description 🐜
+      description: Please provide a clear and concise description of the bug in NextAuth.js
+    validations:
+      required: true   
+  - type: dropdown
+    id: ownproject
+    attributes: 
+      label: Is this a bug in your own project?
+      description: 🚧 – _Do not report bugs with your own project here; ask for help [by raising a question instead](https://github.com/nextauthjs/next-auth/issues/new?assignees=&labels=question&template=question.md) - this helps us a lot with administration overhead._
+      multiple: false
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: true
+  - type: input
+    id: codesandbox
+    attributes:
+      label: How to reproduce ☕️ 
+      description: Please provide a link to a minimal reproduction of the bug
+    validations:
+      required: true 
+  - type: markdown
+    attributes:
+      value: |
+        We encourage you to use one of the templates set up on **CodeSandbox** to reproduce your issue:
+        - [`next-auth-example`](https://codesandbox.io/s/next-auth-example-1kktb)
+        - [`next-auth-typescript-example`](https://codesandbox.io/s/next-auth-typescript-example-se32w)
+
+        🚧 – _If you don't provide any way to reproduce the bug, the issue is at risk of being closed._
+  - type: textarea
+    id: logs
+    attributes: 
+      label: Screenshots / Logs 📽
+      description: We can address the bug you found much faster if you provide contextual screenshots or screen recordings showcasing the issue.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        See [Kap](https://getkap.co/) for a good, easy-to-use, cross-platform screen recording tool.
+  - type: textarea
+    id: environment
+    attributes: 
+      label: Environment 🖥
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        Please run this command in your project's root folder:
+
+        ```sh
+        npx envinfo --system --binaries --browsers --npmPackages "next,next-auth,react"
+        ```
+
+        and paste the output above.
+
+  - type: dropdown
+    id: pr
+    attributes:
+      label: Would you be interested to help solve this bug by opening a PR?
+      multiple: false
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: "## Contributing 🙌🏽"
+  - type: markdown
+    attributes: 
+      value: |
+        It takes a lot of work 🏋🏻‍♀️ maintaining a library like `next-auth`; any contribution is more than welcome 💚
+        In case you're willing to help fix this bug, please let us know here, and we'll reach you 😊 . Otherwise, you can have a look at the issues labelled with [`"good first issue"`](https://github.com/nextauthjs/next-auth/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and pick any of them.      
```

Comment:
```suggestion
  - type: dropdown
    id: pr
    attributes:
      label: Would you be interested to help solve this bug by opening a PR?
      multiple: false
      options:
        - "Yes"
        - "No"
    validations:
      required: true
  - type: markdown
    attributes:
      value: "## Contributing 🙌🏽"
  - type: markdown
    attributes: 
      value: |
        It takes a lot of work 🏋🏻‍♀️ maintaining a library like `next-auth`; any contribution is more than welcome 💚
```

I think the last sentence rather belongs in our `CONTRIBUTING.md`

In addition, my above suggestion would need further tweaking. Is it possible to somehow merge these three sections to one? I think they logically belong together. If not possible, I think the Contributing section should just be a comment, that is not visible in the preview/issue, only when editing.

I would also change the title

> Would you be interested to help solve this bug by opening a PR?

to
 
> ## Contributing 🙌🏽

I am curious about your opinion on this one!

Reply:
> I think the last sentence rather belongs in our CONTRIBUTING.md

I agree, it felt a bit out of place, but I was doing a raw translation of the .md files to the .yaml files, so I left it there.

> Is it possible to somehow merge these three sections to one? I think they logically belong together. 

I believe Github Forms' yaml syntax doesn't allow mixing markdown sections with regular input sections. I actually tried to have the text from the markdown sections in the description field of input, but it did not like it.

> If not possible, I think the Contributing section should just be a comment, that is not visible in the preview/issue, only when editing.

I believe that is indeed better.

Reply:
```yaml
  - type: markdown
    attributes:
      value: "## Contributing 🙌🏽"
  - type: dropdown
    id: pr
    attributes:
      label: Would you be interested to help solve this bug by opening a PR?
      multiple: false
      options:
        - "Yes"
        - "No"
    validations:
      required: true
  - type: markdown
    attributes: 
      value: |
        It takes a lot of work 🏋🏻‍♀️ maintaining a library like `next-auth`; any contribution is more than welcome 💚
```

This renders:
![image](https://user-images.githubusercontent.com/37664921/123894416-f2a67000-d97f-11eb-91a4-365971163c13.png)

Is this better?

Reply:
I just noticed in the [documentation for Github Issue Forms YAML Syntax](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#markdown):

> You can use a markdown element to display Markdown in your form that provides extra context to the user, but is not submitted.